torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19158,10 +19169,20 @@ def __init__(
         "_refs.round",
         torch_opinfo_name="round",
     ),
+    ElementwiseUnaryPythonRefInfo(
+        "_refs.sigmoid",
+        torch_opinfo_name="sigmoid",
+        handles_complex_extremal_values=False,
```

Comment:
Is there an issue we should link to here?

Reply:
Reference: https://github.com/pytorch/pytorch/issues/56012